packages/design-system/src/pages/ docs/Pages.stories.tsx

```diff
@@ -107,8 +107,10 @@ const ItemWithDetails = ({
 						Item {itemId + 1}
 					</ButtonPrimary>
 				}
-				heading={<h3>Item {itemId + 1}</h3>}
-				footer={<ButtonSecondary onClick={() => setVisible(false)}>Close</ButtonSecondary>}
+				header={<h3>Item {itemId + 1}</h3>}
+				footer={(dialog: any) => (
```

Comment:
that's a new component, let's type it strongly :)

Reply:
Done